proto/cosmos/bank/genesis.proto

```diff
@@ -9,23 +9,24 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/bank/types";
 
 // GenesisState defines the bank module's genesis state.
```

Comment:
this is not consistent with auth's comment

Reply:
Yeah true, now all modules have this format for GenesisState.